tests/onnx/quantization/test_classification_models_graph.py

```diff
@@ -41,16 +40,18 @@
                           ]
                          )
 def test_min_max_quantization_graph(tmp_path, model_to_test, model):
-    if model_to_test.model_name == 'shufflenet_v2_x1_0':
-        pytest.skip('While inferencing the model error occured.')
+    if model_to_test.model_name in ['inception_v3', 'googlenet']:
+        pytest.skip('Ticket 96177')
     onnx_model_dir = str(TEST_ROOT.joinpath('onnx', 'data', 'models'))
     onnx_model_path = str(TEST_ROOT.joinpath(onnx_model_dir, model_to_test.model_name))
     if not os.path.isdir(onnx_model_dir):
         os.mkdir(onnx_model_dir)
     x = torch.randn(model_to_test.input_shape, requires_grad=False)
-    torch.onnx.export(model, x, onnx_model_path, opset_version=13)
+    # Ticket 96177
+    torch.onnx.export(model, x, onnx_model_path, opset_version=13, training=torch.onnx.TrainingMode.TRAINING)
 
     original_model = onnx.load(onnx_model_path)
     quantized_model = min_max_quantize_model(model_to_test.input_shape, original_model)
     compare_nncf_graph(quantized_model, model_to_test.path_ref_graph)
-    infer_model(model_to_test.input_shape, quantized_model)
+    # Ticket 96177
+    # infer_model(model_to_test.input_shape, quantized_model)
```

Comment:
Commented code is not allowed. Could you please remove it if it is not needed now?

Reply:
This will be reverted back after the ticket 96177 will be implemented, because of that I commented this

Reply:
Dicussed with @vshampor, he suggested to add explicit comments and it would be enough